modules/installation-aws-tested-machine-types.adoc

```diff
@@ -41,14 +41,20 @@ ifdef::localzone[]
 .Machine types based on x86_64 architecture for AWS Local Zones
 [%collapsible]
 ====
-* `c5.2xlarge`
-* `c5d.2xlarge`
-* `m5.xlarge`
-* `m5.2xlarge`
-* `t3.xlarge`
+* c5.*
+* c5d.*
+* m6i.*
+* m5.*
+* r5.*
+* t3.*
+====
+
+[NOTE]
+====
+If you want to use the `t3` instance type, you must manually specify it in the `install-config.yaml` file. 
```

Comment:
This may make users confused: what about others types? What if the user wants to use `c5d.2xlarge` in the LZ, which supports both `m5.2xlarge` and `c5d.2xlarge`? User still needs to provide instance type in the config, so I suggest removing this note entirely, wdyt?

Reply:
+1